File-Formats/DocIO/Charts/Chart-Title.md

```diff
@@ -196,4 +196,109 @@ End Using
+        chart.ChartData.SetValue(2, 2, 141.396);
+        chart.ChartData.SetValue(3, 1, "Stanley Hudson");
+        chart.ChartData.SetValue(3, 2, 80.368);
+        //Creates a new chart series with the name “Sales”.
+        IOfficeChartSerie pieSeries = chart.Series.Add("Sales");
+        pieSeries.Values = chart.ChartData[2, 2, 3, 2];
+        //Sets category labels.
+        chart.PrimaryCategoryAxis.CategoryLabels = chart.ChartData[2, 1, 3, 1];
+        //Saves the Word document to MemoryStream.
+        MemoryStream stream = new MemoryStream();
+        document.Save(stream, FormatType.Docx);
+        //Closes the document.
+        document.Close();
+}
+{% endhighlight %}
+
+{% highlight c# tabtitle="C# [Windows-specific]" %}
+using (WordDocument document = new WordDocument("Template.docx", FormatType.Docx))
+{
+    //Adds section to the document.
+    IWSection sec = document.AddSection();
+    //Adds paragraph to the section.
+    IWParagraph paragraph = sec.AddParagraph();
+    //Creates and Appends chart to the paragraph.
+    WChart chart = paragraph.AppendChart(446, 270);
+    //Sets chart type.
+    chart.ChartType = OfficeChartType.Pie;
+    //Sets chart title.
+    chart.ChartTitle = string.Empty;
+    //Sets data for chart.
+    chart.ChartData.SetValue(1, 1, "");
+    chart.ChartData.SetValue(1, 2, "Sales");
+    chart.ChartData.SetValue(2, 1, "Phyllis Lapin");
+    chart.ChartData.SetValue(2, 2, 141.396);
+    chart.ChartData.SetValue(3, 1, "Stanley Hudson");
+    chart.ChartData.SetValue(3, 2, 80.368);
+    //Creates a new chart series with the name “Sales”.
+    IOfficeChartSerie pieSeries = chart.Series.Add("Sales");
+    pieSeries.Values = chart.ChartData[2, 2, 3, 2];
+    //Sets category labels.
+    chart.PrimaryCategoryAxis.CategoryLabels = chart.ChartData[2, 1, 3, 1];
+    //Saves the document
+    document.Save("Sample.docx");
+}
+{% endhighlight %}
+
+{% highlight vb.net tabtitle="VB.NET [Windows-specific]" %}
+'Adds section to the document.
+ Dim sec As IWSection =  document.AddSection() 
+'Adds paragraph to the section.
+ Dim paragraph As IWParagraph =  sec.AddParagraph() 
+'Creates and Appends chart to the paragraph.
+ Dim chart As WChart =  paragraph.AppendChart(446,270) 
+'Sets chart type.
+ chart.ChartType = OfficeChartType.Pie
+'Sets chart title.
+ chart.ChartTitle = String.Empty
+'Sets data for chart.
+ chart.ChartData.SetValue(1, 1, "")
+ chart.ChartData.SetValue(1, 2, "Sales")
+ chart.ChartData.SetValue(2, 1, "Phyllis Lapin")
+ chart.ChartData.SetValue(2, 2, 141.396)
+ chart.ChartData.SetValue(3, 1, "Stanley Hudson")
+ chart.ChartData.SetValue(3, 2, 80.368)
+'Creates a new chart series with the name “Sales”.
+ Dim pieSeries As IOfficeChartSerie =  chart.Series.Add("Sales") 
+ pieSeries.Values = chart.ChartData(2, 2, 3, 2)
+'Sets category labels.
+ chart.PrimaryCategoryAxis.CategoryLabels = chart.ChartData(2, 1, 3, 1)
+'Creates file stream.
+ Imports (MemoryStream stream = New MemoryStream())
```

Comment:
Please save in path, for VB as mentioned in earlier feedback.

Reply:
Addressed the feedback